docs/user/openstack/provider_networks.md

```diff
@@ -0,0 +1,120 @@
+    to install OpenShift (from the clouds.yaml) and we don't want them to be owned by `admin` otherwise
+    it'll cause Terraform to fail creating the ports.
+
+    Example where our project name is `openshift`:
+
+        openstack network create --project openshift (...)
+        openstack subnet create --project openshift (...)
+
+    More information can be found about how to create provider networks [here][5].
+
+* The Metadata IP has to be routed so the bootstrap nodes can fetch the required information
+  to provision the node and the cluster. The Neutron DHCP server also has to be enabled.
+
+  This can be achieved by specifying a route when creating the subnet:
+
+        openstack subnet create --dhcp --host-route destination=169.254.169.254/32,gateway=$ROUTER_IP" (...)
+
+  (where the gateway is the IP of the router in charge of routing the traffic of the Provider Network).
+
+**Note:** This should not apply to OVN because Metadata is completely distributed across OpenStack compute nodes
+          and served from the ovnmeta-xxxxx-xxxx namespace. Therefore, there is nothing to do.
+
+* When deployment baremetal workers on provider networks with OVN, the subnet should be created with DHCP enabled
+  and the Neutron DHCP agent should be deployed and running. There is an [ongoing effort][6] to remove that
+  dependency.
+
+        openstack subnet create --dhcp
+
+
+## Deploying cluster with primary interface on a provider network with IPI
+
+
+- Considerations: make sure all prerequisites documented previously have been met.
+
+- Create install-config.yaml:
+
+    - Set `platform.openstack.apiVIP` to the IP address for the API VIP.
+    - Set `platform.openstack.ingressVIP` to the IP address for the Ingress VIP.
+    - Set `platform.openstack.machinesSubnet` to the subnet ID of the provider network subnet.
+    - Set `networking.machineNetwork.cidr` to the CIDR of the provider network subnet.
+
+**Note:**
+
+`platform.openstack.apiVIP` and `platform.openstack.ingressVIP` both need to be an unnassigned IP
+address on the `networking.machineNetwork.cidr`.
+
+    Example:
+
+        (...)
+        platform:
+          openstack:
+            apiVIP: <IP address in the provider network reserved for the API VIP>
+            ingressVIP: <IP address in the provider network reserved for the Ingress VIP>
+            machinesSubnet: <provider network subnet ID>
+            (...)
+        networking:
+          machineNetwork:
+          - cidr: <provider network subnet CIDR>
+
+- Run the openshift installer:
+
+      ./openshift-install create cluster --log-level debug
+
+- wait for the installer to complete.
+
+
+## Known issues
+
+* Terraform fails to create ports with permissions issue, see requirements.
+
+* Metadata can't be reached (routing issue), see requirements documented earlier.
+
+* DHCP traffic is not supported on provider networks using OVN when doing baremetal, see
```

Comment:
```suggestion
* DHCP traffic is not supported on provider networks using OVN when doing baremetal, see
```

I'm not sure of the value of these troubleshooting bullets, yet. We don't have example messages for a quick ctrl+F, and the only suggested course of action is to check the prereqs.